### ISSUE TITLE
Port over User Creation Wizard

### ISSUE THREAD

**Issue by rdig** [JoinColony/colonyCDapp#31]
- Port over the components
- Should live in `components/common`

~~Note that this is blocked by #30~~

**Comment by rdig** [comment]
Note that this will no longer send a transaction to the chain, as we don't need it anymore. 

- We'll be registering the username in db only, so only a mutation will be needed. 
- We need to remove all the user creation sagas.
- mutation will be sent directly from the component

**Comment by rdig** [comment]
Also note that this issue is a bit larger that originally it may appear since it will also cover the user onboard design refresh, so it will most likely need to be treated as a big feature and split up into multiple issues.

Whoever gets to it, please have a call w/ @arrenv so he can hand over the design specs

_I'll leave a screenshot so you're aware of what this is all about:_

![Screenshot from 2022-10-31 14-33-52](https://user-images.githubusercontent.com/1193222/199008898-b1bbc7ae-0bfe-4daf-be0f-66123d78e366.png)

**Comment by arrenv** [comment]
The design is being cleaned up, now that Mel is on full-time. So, depending on when this get picked up, it may need to use the old design, which we also have specs for here - https://www.figma.com/file/xGRNwVyii0qaCkWs4hASUn/Onboarding-2.0?node-id=757%3A13573

**Comment by willm30** [comment]
I've discussed the scope of this issue with @arrenv . I will be adding the user email screen to the wizard per [this comment](https://github.com/JoinColony/colonyCDapp/issues/32#issuecomment-1305366825) but leaving the rest of the design revamp for a separate issue.
